lib/active_merchant/billing/gateways/payeezy.rb

```diff
@@ -119,27 +135,29 @@ def add_authorization_info(params, authorization)
         params[:method] = method
       end
 
+      def add_creditcard_for_tokenization(params, payment_method)
+        params[:apikey] = PARAMS_FOR_TOKENIZATION[:api_key]
+        params[:js_security_key] = PARAMS_FOR_TOKENIZATION[:js_security_key]
+        params[:ta_token] = PARAMS_FOR_TOKENIZATION[:ta_token]
+        params[:callback] = PARAMS_FOR_TOKENIZATION[:callback]
+        params[:type] = PARAMS_FOR_TOKENIZATION[:type]
```

Comment:
Payeezy docs make it seem like the `ta_token`, `callback`, and `type` fields are static? Correct me if I'm wrong 😬 If they are, then we can just hardcode those inline vs reference them in a hash.

Reply:
👍